### ISSUE TITLE
Add visual focus indication for Clarity Accordion / Stack View 

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#3974]
## Summary

Accordions  do not have a clear visual indication of focus. For accordions that are collapsed, visual focus is indicated using a slight background color change. For accordions that are expanded, no visual focus indicator is available.

* What is the change?
Provide visual indication that a section of the component is focused 
* Why should it go in Clarity?
To provide basic accessibility features
* Does this change impact existing behaviors? If so how?
No
* If this change introduces a new behavior, is this behavior accessible?
Yes

## Use case

As a user of the component I want to have visual indication with a proper contrast ration which is the focused section of the component

## Examples

No

## Workarounds

No

**Comment by gnomeontherun** [comment]
Accordions do have a focus indication, see the behavior below as well as review the demo here https://clarity.design/documentation/accordion.

![image](https://user-images.githubusercontent.com/79500/68145983-9fef8900-fefc-11e9-8255-2bf13df7b5da.png)

Without a demo, it is hard to know if there is some other issue you are facing or what version of Clarity you are using, but the Accordion has been vetted by our accessibility team. If you can point to a reproduction with a demo, please report back. Otherwise, I'm not sure exactly what the issue here is.

**Comment by bashibozuk** [comment]
HI @gnomeontherun. This is true about the accordion but not for the Stack View. Providing screenshots.
![Screenshot 2019-11-06 17 56 18](https://user-images.githubusercontent.com/2686283/68314611-1f2cba80-00bf-11ea-90b7-0eac416468f6.png)
![Screenshot 2019-11-06 17 56 32](https://user-images.githubusercontent.com/2686283/68314612-1f2cba80-00bf-11ea-9e6a-57e0a4d3fa09.png)

**Comment by bashibozuk** [comment]
@mathisscott I updated the description and content to point to Stack View which is actually the problematic component. As shown on the picture posted in my previous comment, answering to your reproduction ask the stack-block(below) does not have the blue outline which indicates the focused header of the accordion(above).
The screen shot is taken from the official clarity documentation site and is currently reproducible there with the latest versions.
The problems that this issue is about are 2:

- Focus indication in StackView and Accordion component headers is not consistent, which is at least confusing for the user 
- The StackView lacks focus( and hover) indication on expanded sections, but provides them for collapsed section which is again inconsistent
